### ISSUE TITLE
Update - first cut with training (architecture needs redoing but...)  "apricot-mountain-66" 

### ISSUE THREAD

**Issue by johndpope** [johndpope/implicit-motion-function#5]
was cross checking the paper an hour ago - then found this
https://openaccess.thecvf.com/content/CVPR2024/supplemental/Gao_Implicit_Motion_Function_CVPR_2024_supplemental.pdf

i make progress with my own direction without - but now all the code needs redoing....
this branch doesn't align to paper (not using GAN architecture) - but trains and converges...
.- just used simple lpips loss - i was really happy with the results 
https://github.com/johndpope/IMF/tree/v1
it's missing the positional embeddings in the implicit motion alignment module - 
but seems to work...(though there's gradient explosion / vanishing) - i bandaid the models with batch norm and that helped.
https://wandb.ai/snoozie/IMF/runs/f9o9vvje?nw=nwusersnoozie

![Screenshot from 2024-07-29 05-48-38](https://github.com/user-attachments/assets/21565e45-3feb-46d3-a605-8a7234cde0cd)

in the videodataset - it's just using the skipframe 1 (the next frame) - this needs playing around with.

git checkout -b "apricot-mountain-66" f796a3712c9810d6c269794603f7db000fad5769
